roofit/roofit/inc/RooBernstein.h

```diff
@@ -33,17 +34,19 @@ class RooBernstein : public RooAbsPdf {
 
   Int_t getAnalyticalIntegral(RooArgSet& allVars, RooArgSet& analVars, const char* rangeName=0) const ;
   Double_t analyticalIntegral(Int_t code, const char* rangeName=0) const ;
+  void selectNormalizationRange(const char* rangeName=0, Bool_t force=kFALSE) ;
 
 private:
-
-  RooRealProxy _x;
+  
+  RooTemplateProxy<RooRealVar> _x ;
```

Comment:
With this, the Bernstein requires `_x` to be a `RooRealVar`. In this case, the constructor should also only take a `RooRealVar&`. I suspect, though, that you could also live with `RooLinearVar` (linear transformation of a `RooRealVar`). In that case, you can widen both constructor and proxy to take `RooAbsRealLValue&`.

Reply:
Hi, this works, thanks !